src/metadata-server/src/main/java/org/apache/kylin/rest/controller/NUserController.java

```diff
@@ -478,6 +482,7 @@ public EnvelopeResponse<String> updateUserPassword(@RequestBody PasswordChangeRe
     @ResponseBody
     public EnvelopeResponse<UserDetails> authenticate() {
         EnvelopeResponse<UserDetails> response = authenticatedUser();
+        checkSessionStoreType(KylinConfig.getInstanceFromEnv());
```

Comment:
When we change the spring-session version from kylin special version to community version, is this ok?

Reply:
it works too.

Reply:
this issue is caused by store-type override, so if changed the version,  the override issue still exist.